ui/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessTypesForCategories.scala

```diff
@@ -9,7 +9,7 @@ import scala.collection.JavaConverters._
 import net.ceedubs.ficus.Ficus._
 
 
-class ProcessTypesForCategories(config: Config) {
+case class ProcessTypesForCategories(config: Config) {
```

Comment:
Why it is case class? It is now a bean that provide publicly `config` and will be somewhere serialized with underlying `config`?

Reply:
true that, we don't need case class here

Reply:
done